src/lib/Components/Inbox/Conversations/ConversationSnippet.tsx

```diff
@@ -196,24 +225,14 @@ export default Relay.createContainer(ConversationSnippet, {
 interface RelayProps {
   conversation: {
     id: string | null
-    from: {
-      name: string
-      email: string
-    }
     to: {
       name: string
     }
     last_message: string
     last_message_at: string | null
-    artworks: Array<{
-      id: string
-      href: string | null
-      title: string | null
-      date: string | null
-      artist_names: string | null
-      image: {
-        url: string | null
-      } | null
+    __typename: string
+    items: Array<{
+      item: any
     } | null> | null
```

Comment:
I’d be interested to see what it would output for a union field at all, I don’t think I covered that yet :)